.github/workflows/publish.yml

```diff
@@ -64,3 +64,45 @@ jobs:
           python -m pip install --upgrade pip
           pip install python-doctr
           python -c "import doctr; print(doctr.__version__)"
+
+  conda:
+    if: "!github.event.release.prerelease"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v3
+        with:
+          auto-update-conda: true
+          python-version: 3.8
+      - name: Install dependencies
+        shell: bash -el {0}
+        run: conda install -y conda-build conda-verify anaconda-client
+      - name: Get release tag
+        id: release_tag
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+      - name: Build and publish
+        shell: bash -el {0}
+        env:
+          ANACONDA_API_TOKEN: ${{ secrets.ANACONDA_TOKEN }}
+          VERSION: ${{ steps.release_tag.outputs.VERSION }}
+        run: |
+          echo "BUILD_VERSION=${VERSION}" >> $GITHUB_ENV
+          python setup.py sdist
```

Comment:
calling `setup.py` directly is deprecated, you'll want something like `python -m build --sdist` instead
see also https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html

Reply:
@frgfm  ? :)

Reply:
Oh I didn't know, thanks mara!
But for this PR, I don't think we need the edit:
- we'd have to do this for the other occurrences in the repo (feels like a standalone PR to me, and we should check backward compatibility)
- it will be deprecated but it's still working to the best of my understanding right?

Reply:
> it will be deprecated but it's still working to the best of my understanding right?

Yes, for the time being. They might want to remove it after some deprecation period, but I don't know if they'll ever do, as this somehow doesn't seem to have reached the crowd at large.